src/sidebar/services/annotations.ts

```diff
@@ -70,21 +70,26 @@ export class AnnotationsService {
       this._store.defaultAuthority();
     const mentionsEnabled = this._store.isFeatureEnabled('at_mentions');
 
-    if (draft) {
-      const textWrapper = !mentionsEnabled
-        ? (text: string) => text
-        : mentionsOptions.mentionMode === 'display-name'
-          ? (text: string) =>
-              wrapDisplayNameMentions(text, mentionsOptions.usersMap)
-          : (text: string) => wrapMentions(text, authority);
-
-      changes.tags = draft.tags;
-      changes.text = textWrapper(draft.text);
-      changes.permissions = draft.isPrivate
-        ? privatePermissions(annotation.user)
-        : sharedPermissions(annotation.user, annotation.group);
+    if (!draft) {
+      return { ...annotation };
```

Comment:
I'm not entirely sure if we need to return a new instance here, but I tried to preserve the previous behavior just in case.